src/scss/patterns/_offset-grid.scss

```diff
@@ -43,3 +43,24 @@ $break: 'wide-break';
     padding-left: 0;
   }
 }
+
+//services page
+
+.offset-media {
+  --offset-media-padding: 0;
+  display: block;
+  padding: var(--offset-media-padding);
+
+  &:nth-child(odd) {
+    padding: var(--offset-media-padding);
+  }
+
+  &:nth-child(even) {
+    padding: var(--offset-media-padding);
+  }
```

Comment:
This feels like a lot of repetition to override the padding set in `layout.clip-base()` and `layout.clip()` mixins. Might be cleaner if we setup those mixins to allow a custom-property override? Or maybe we could override the existing `--clip-space` (I haven't looked closely at how that one works)?

Reply:
I made some tweaks to the custom properties in the layout mixins. These styles are related to the `offset-item` class too. Let me know if you think what I've done has simplified things. 

Still working on some of the rest of the review edits. Glad you like how it's going so far!